include/nuttx/fs/ioctl.h

```diff
@@ -180,6 +180,9 @@
 #define FIONCLEX        _FIOC(0x000e)     /* IN:  None
                                            * OUT: None
                                            */
+#define FIOC_FILEPATH   _FIOC(0x0004)     /* IN:  FAR char *(length >= PATH_MAX)
```

Comment:
```suggestion
#define FIOC_FILEPATH   _FIOC(0x000f)     /* IN:  FAR char *(length >= PATH_MAX)
```
This value will conflict `FIOC_FILENAME`.

Reply:
Done.